TEI.js

```diff
@@ -446,10 +451,14 @@ function generateItem(item, teiDoc) {
 		imprint.appendChild(imprintDate);
 	}
 
-	// flag unpublished if there is no date | publisher | place
+	// flag unpublished if there is no date | publisher | place by creating empty elements date | publisher | pubPlace
```

Comment:
this doesn't work as is: it creates an empty node for all three if any one of them doesn't exist. Why not instead just tag this on as `else` conditions individually for `if (item.date)` etc.?

Reply:
As a matter of fact, it did not work that way and in the previous code neither (while the pipes should have said OR, it was correct for all items with at least one of the field). Following your proposition and given that only one field is needed, I've added "!(item.date)" => empty date element.
And I've added some lines to keep the website title (webpage as analytic